python/shipDigiReco.py

```
@@ -146,6 +186,340 @@ def reconstruct(self) -> None:
+            indices_vector = ROOT.std.vector("unsigned int")()
+            for index in hit_indices:
+                if 0 <= index < len(self.strawHitToDigi):
+                    indices_vector.push_back(self.strawHitToDigi[index])
+
+            aTracklet = ROOT.Tracklet(1, indices_vector)
+            self.fTrackletsArray.push_back(aTracklet)
+
+        for vtx in vertices:
+            try:
+                input_addr = int(vtx._extracted_params_addr) if hasattr(vtx, "_extracted_params_addr") else 0
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)